bears/typescript/TSLintBear.py

```diff
@@ -17,7 +17,8 @@ class TSLintBear:
     """
 
     LANGUAGES = {'TypeScript'}
-    REQUIREMENTS = {NpmRequirement('tslint', '3')}
+    REQUIREMENTS = {NpmRequirement('tslint', '3'),
+                    NpmRequirement('typescript', '1.7.3')}
```

Comment:
I think you can use '>=1.7.3' here, and update the generator to detect this and omit the `~`.